### DISCUSSION TITLE
Merging multiple accounts into 1 stats page

### DISCUSSION THREAD

**Discussion by JobvanTuijl** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#713] [Ideas]
With the current statistics page, it only shows for 1 twitch account. It would be nice if I could merge multiple accounts into 1 page where for example for streamer1 I can see all the graphs of all my twitch accounts overlayed.

**Comment by XaaRii** [comment]
This repository is not maintained anymore. Please check out (the currently active) fork from rdavydov https://github.com/rdavydov/Twitch-Channel-Points-Miner-v2

You can suggest your ideas there.
